migrations/migration-9/pin-set.js

```diff
@@ -107,10 +103,10 @@ async function * walkItems (blockstore, node) {
       // if a fanout bin is not 'empty', dig into and walk its DAGLinks
       const linkHash = link.Hash
 
-      if (!uint8ArrayEquals(EMPTY_KEY, linkHash.bytes)) {
+      if (!EMPTY_KEY.equals(linkHash)) {
```

Comment:
do we need to be concerned about the CIDv1 form of EMPTY_KEY? can we get here with a v1? https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-pb.md#zero-length-blocks

Reply:
if so, that'd still be a problem with the current form - `1220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` for a v0 and `01701220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` for a v1, so even just comparing `.bytes` they're different.

Reply:
No, the pinset dag has only ever been v0.